starwars/src/components/Character/SWCharacters.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import './StarWars.css';
+
+class Character extends Component {
+    constructor() {
+        super();
+        this.state = {
+
+        }
+
+    }
+
+    componentDidMount() {
+        this.setState({names: this.props.datum.name})
```

Comment:
This is not actually populating your state.  There are a few issues here: (1) there isn't a `names` key on your state object, (2) there is a disconnect between the `names` naming conventions and the value `this.props.datum.name` which you are assigning to it. You're essentially implying `names: character-name`. Since you can already reach the character's name by props, why use state at all?